hololive-beat/hololive-beat/src/app/music_select_page/music_carousel.rs

```diff
@@ -105,43 +106,52 @@ impl Component for MusicCarousel<'_> {
 
 #[component]
 struct MusicCard<'a> {
-    pub wh: Wh<Px>,
-    pub center_xy: Xy<Px>,
-    pub rotate: Angle,
+    pub music_card_wh: Wh<Px>,
+    // offset -2.5 ~ 2.5
```

Comment:
use `///` instead `//`. /// is doc so you can see that comment on vscode intellisense